src/NativeScript/GlobalObject.mm

```diff
@@ -118,6 +119,27 @@ static void microtaskRunLoopSourcePerformWork(void* context) {
     self->drainMicrotasks();
 }
 
+static void runLoopBeforeWaitingPerformWork(CFRunLoopObserverRef observer, CFRunLoopActivity activity, void* info) {
+    GlobalObject* self = static_cast<GlobalObject*>(info);
+    VMEntryScope* currentEntryScope = self->vm().entryScope;
+    if (self->vm().topCallFrame && currentEntryScope && !currentEntryScope->didPopListeners().isEmpty()) {
+        FFIFunctionCall* function_call = static_cast<FFIFunctionCall*>(self->vm().topCallFrame->callee());
+        const Meta* meta = Metadata::MetaFile::instance()->globalTable()->findMeta(WTF::ASCIILiteral("UIApplicationMain"));
```

Comment:
👍 
PS: `findMeta` can be called with pure `char *`, no need to construct `ASCIILiteral`.